src/sagemaker/fw_utils.py

```diff
@@ -113,7 +113,22 @@ def _using_merged_images(region, framework, py_version, accelerator_type, framew
     is_gov_region = region in VALID_ACCOUNTS_BY_REGION
     is_py3 = py_version == "py3" or py_version is None
     is_merged_versions = _is_merged_versions(framework, framework_version)
-    return (not is_gov_region) and is_merged_versions and is_py3 and accelerator_type is None
+    return (
+        (not is_gov_region)
+        and is_merged_versions
+        and (is_py3 or _is_tf_14(framework, framework_version))
```

Comment:
should this be _is_not_tf_13? because we do expect TF 1.15 py2 as well

Reply:
As discussed offline, we will change this method to _is_tf_14_or_later